### ISSUE TITLE
[Enhancement]: Allow Mini-Cart Pattern to be inserted in any template

### ISSUE THREAD

**Issue by gmatta01** [woothemes/woocommerce#50870]
### Describe the solution you'd like

Introducing an **always-visible Mini-Cart** would be a valuable addition to the block library, providing a seamless shopping experience for various types of online stores. This enhancement would allow the Mini-Cart pattern to be inserted and displayed in any part of a template, such as a sidebars or custom panels.

### Use Cases

1. **Food Ordering Shops**:
   In food ordering websites, particularly on desktop views, it's crucial to have the cart always visible on the right sidebar. This setup allows customers to add optional items without leaving the current view, streamlining the ordering process. By reducing unnecessary clicks and page navigation, users can quickly move to the checkout, enhancing the overall user experience.

2. **Quick-Order Websites (e.g., Groceries)**:
   Grocery stores and other quick-order websites can also benefit from an always-visible Mini-Cart. These platforms often focus on speed and convenience, where customers expect to make rapid purchases. Having the Mini-Cart accessible at all times allows users to review and adjust their selections in real-time, making the checkout process more efficient.

### Benefits

- **Enhanced User Experience**: By reducing the steps needed to review and modify the cart, the overall shopping experience is smoother and more intuitive.
- **Increased Conversions**: An always-visible cart can lead to higher conversion rates as users are constantly reminded of their selected items, encouraging them to proceed to checkout.
- **Customization Flexibility**: Allowing the Mini-Cart to be placed in any part of the template provides flexibility for theme developers and store owners to design their site layouts including Mini-cart pattern based on their specific needs.

**Comment by mikejolley** [comment]
Seems like something good we could consider since the block already exists. CC @opr for planning. Will add to the backlog.

**Comment by gmatta01** [comment]
Any update on this feature?
Appreciate your inputs.

**Comment by opr** [comment]
Hey @gmatta01 no updates to share currently, but we will provide information on this issue when we address it. Thanks!

**Comment by pierorocca** [comment]
@gmatta01 thanks so much for the detailed use cases and benefits analysis. Could you provide a few examples you could share of popular sites that use this pattern?

**Comment by gmatta01** [comment]
Hey @pierorocca!
Here are few popular examples: 

- https://order.storekit.com/slice-slinger-s/menu
- https://deliveroo.co.uk/menu/London/london-bridge/mcdonalds-1542-london-bridge/?day=today&geohash=gcpvjcnm9kbj&time=ASAP

Letting customers see their cart anytime is super handy, especially for things like food orders, groceries, or other quick delivery items. With one click, they can jump straight to checkout, making the whole process easy and fast.  

A lot of restaurants already use third-party food delivery apps, but adding their own WooCommerce store with pickup (and even delivery) options would be an awesome upgrade. It’s a great way to offer more convenience and keep more control over orders.

Having a sidebar cart always visible that is already available On-Click, would be quiet useful IMO.

**Comment by pierorocca** [comment]
Thanks @gmatta01 I'm having a look and have shared these with our designer who is currently taking a holistic look at the current and future state of the mini-cart. This is good timing.

**Comment by pierorocca** [comment]
I could see how a food order and any other type of order where the user is building up an order with many items and probably for multiple people, would benefit. The accuracy and attention to detail is elevated and avoiding the mini-cart from opening and closing could be beneficial.

What's your vision for this on mobile @gmatta01 ?

**Comment by gmatta01** [comment]
Yes, it would be super handy! Like a mobile app, a website built with a block builder can deliver a much faster response.  

A simple JS event to trigger the cart display on mobile is all needed. Developers can fully customize the experience and decide when to show the cart.  

For example, on mobile, I’d add a bottom bar showing the total amount and number of items. When clicked, it could display the cart, just like it does now. I’d also include a button for categories that, when clicked, slides to their respective products for a smooth, user-friendly experience.

**Comment by pierorocca** [comment]
Awesome thanks again @gmatta01. As noted we're taking a holistic look at the mini-cart and have passed on your use cases for consideration. Desktop seems like the most simple as we'd just not close the drawer/side cart. Mobile is a bit more involved. Your suggestion is a good one.
